tools/releaseBuild/azureDevOps/templates/upload.yml

```diff
@@ -66,3 +66,9 @@ steps:
     ContainerName: '$(AzureVersion)-private'
     resourceGroup: '$(StorageResourceGroup)'
   condition: and(succeeded(), eq('${{ parameters.msix }}', 'yes'), eq(variables['SHOULD_SIGN'], 'true'))
+
+- template: upload-final-results.yml
```

Comment:
This is a build template.  This can't be used in a release.

Reply:
You need to use this task in a release pipeline
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact?view=azure-devops


The azure tasks would be the same.